### ISSUE TITLE
Col width, col visibility and grid layout

### ISSUE THREAD

**Issue by cjohnsonuk** [koalyptus/TableFilter#599]
Summary : Col widths don't work for table header row if grid layout is enabled and col visibility is used to hide columns.
Affects : TableFilter v0.6.35

example: If I define col widths and set the view to grid layout then when I hide columns the col widths in the table headers don't follow the col widths in the table body.  Eg on this page

http://koalyptus.github.io/TableFilter/columns-visibility.html

run this script

```
tf.destroy();
var tfConfig = {
        base_path: 'tablefilter/',
        paging: true, col_widths: ['50px', '200px', '100px', '50px','100px','500px'],  grid_layout: {
                    width: '100%',
                    height: '350px'
                } ,
        state: { types: ['cookie'] },
        alternate_rows: true,
        rows_counter: true,
        btn_reset: {
            text: 'Clear'
        },
        status_bar: true,
        col_1: 'select',
        col_2: 'select',
        col_types: [
            'number', 'string', 'string',
            'formatted-number', 'formatted-number', 'string'
        ],
on_filters_loaded: function(tf){
                    var dom = tf.gridLayout ?
                        tf.feature('gridLayout').headTbl : tf.dom();

                    // hide the filters row
                    dom.rows[tf.getFiltersRowIndex()].style.display = 'none';
                },
        /* columns visibility and sort extension */
        extensions: [{
              name: 'colsVisibility',
              at_start: [3, 5],
              text: 'Columns: ',
              enable_tick_all: true
          }, {
              name: 'sort'
        }]
    };
    var tf = new TableFilter('demo', tfConfig);
    tf.init();
```
and then hide some columns.

Table body col widths appear to keep the correct widths based on the actual data but the table header columns' widths remain constant for their visible position, not their absolute position in the table.  So table header column text moves from wide columns to narrow columns and vice versa as columns are shown/ hidden with the col widths in the header remaining constant in the header section. Better to take a look and see than me try to explain it ;-)

**Comment by koalyptus** [comment]
@cjohnsonuk tx for pointing out this bug, should now be fixed on v0.6.37, below a demo with your configuration: 
https://codepen.io/koalyptus/pen/ZomqrV

**Comment by cjohnsonuk** [comment]
Wow! That was quick! Thank you so much!
